dimagi_lake/aggregation/utils.py

```diff
@@ -0,0 +1,24 @@
+from consts import ORG_STRUCTURE
+
+
+def get_location_group_by_columns(location_level, loc_info_suffix):
```

Comment:
nit: doc string and/or test would help explain what this does

Reply:
doc string to add 
```
"""
Returns the list of location columns which are to be added in the group by list based on the location level. For Example:
Location_level=2 and loc_info_suffix=['name', 'id'] would return ['state_id', 'state_name', 'district_id', 'district_name']
"""
```